cli/sawtooth_cli/cluster.py

```diff
@@ -66,7 +67,18 @@ def add_cluster_status_parser(subparsers, parent_parser):
 
 
 def add_cluster_start_parser(subparsers, parent_parser):
-    parser = subparsers.add_parser('start', parents=[parent_parser])
+
+    epilog = '''details:
+    The available transaction processors specified by -P depend upon the
+    method (specified by -m). When using docker, the included processors
+    can be found using the command 'docker images', and include the string
+    tp_ within the image name.
```

Comment:
The tp_ prefix is our convention but not a requirement.  Suggest we omit the content after the last comma.